.pyproject_generation/pyproject_custom.toml

```
@@ -1,11 +1,11 @@
 [project]
 name = "metldata"
-version = "1.2.0"
+version = "1.2.1"
 description = "metldata - A framework for handling metadata based on ETL, CQRS, and event sourcing."
 dependencies = [
-    "hexkit[akafka,mongodb]~=1.0.0",
-    "ghga-service-commons[api,auth]~=1.1.0",
-    "ghga-event-schemas~=1.0.0",
+    "hexkit[akafka,mongodb]~=2.1.1",
+    "ghga-service-commons[api,auth]~=2.0.1",
+    "ghga-event-schemas~=3.0.0",
```

Please have a look at our versioning guidelines: https://ghga.pages.hzdr.de/internal.ghga.de/main/developer/developer_sops/dsop012_python_dependency_management/#dependency-versioning-guidelines
```suggestion
    "hexkit[akafka,mongodb] >=2.1.1, <3",
    "ghga-service-commons[api,auth] >=2.0.1, <3",
    "ghga-event-schemas >=3.0.0, <4",
```

Linkml dependencies are an exception here as they are very unstable and are thus pinned.